cardano-api/src/Cardano/Api/CBOR.hs

```diff
@@ -6,6 +8,11 @@ module Cardano.Api.CBOR
   , publicKeyFromCBOR
   , publicKeyToCBOR
 
+  , decodeShelleyVerificationKey
+  , encodeShelleyVerificationKey
```

Comment:
Probably do not need to export the `encode `and `decode` functions.

Reply:
Sure, I'll drop them from the export list.